content/master/concepts/providers.md

```diff
@@ -22,7 +22,7 @@ Examples of providers include:
 * [Provider Kubernetes](https://github.com/crossplane-contrib/provider-kubernetes)
 
 {{< hint "tip" >}}
-Find more providers in Crossplane's [public package registries](https://www.crossplane.io/registries).
+Find more providers in Crossplane's [public package registries](https://marketplace.upbound.io/).
```

Comment:
great catch @matmilbury! We should actually remove this link entirely. For vendor neutrality, we don't want to link directly to the upbound marketplace at all. 

There's a similar hit that should be also entirely removed at https://github.com/crossplane/docs/blob/master/content/master/getting-started/introduction.md#providers

Would you be able to make these changes across all the version folders in the `content` dir? i.e. v1.17, v1.18, v1.19, and master? That would be really helpful! 🙇‍♂️

Reply:
@matmilbury someone brought this up in Crossplane slack again today, so I went ahead and pushed a commit to your branch that fixes all the locations, and kept you as the commit author 😉